portal/templates/instance_profile.html

```diff
@@ -114,5 +125,22 @@ <h1>Detailed services information currently not available</h1>
     }
 }
 
+var instance = $('#instance-data').data();
+var instance_id = instance.name
+
+var logtest = new Vue({
```

Comment:
create a Vue instance which has the getLogs function inside. getLogs function is invoked by button click